### PR TITLE
LRDOCS-9488 Translating Web Content Wordsmithed

### PR DESCRIPTION
**Pull request by sez11a**
https://issues.liferay.com/browse/LRDOCS-9488

@JamesAGarcia

- LRDOCS-9488 adds Automatically Translating Web Content article w/imgs
- LRDOCS-9488 adds article to rst and landing page, adds additional info links
- LRDOCS-9488 wordsmithing rst
- LRDOCS-9488 Wordsmithing.

### CODE REVIEW

File: docs/dxp/latest/en/content-authoring-and-management/web-content/translating-web-content/automatically-translating-web-content.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Automatically Translating Web Content
+
+Liferay integrates with the Google Cloud Translation API to generate automatic translations of Web Content. To enable this feature, you must have an active Google Cloud project with Cloud Translation API enabled. This project must also have a service account with a JSON account key. See official [Google Cloud Translation documentation](https://cloud.google.com/translate/docs/setup) for more information.
+
+## Enabling Automatic Translation
+
+1. Open the *Global Menu*(![Global Menu]()). Click the *Control Panel* tab and go to *System Settings* &rarr; *Translation*.
```
Comment (jrhoun) [inline docs/dxp/latest/en/content-authoring-and-management/web-content/translating-web-content/automatically-translating-web-content.md:7]: @JamesAGarcia Missing icon.

File: docs/dxp/latest/en/content-authoring-and-management/web-content/translating-web-content/automatically-translating-web-content.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Using Automatic Translation
+
+Once Google Cloud Translation is enabled, auto translate buttons are added to the translation interface for Web Content.
+
+Follow these steps to automatically translate all or only individual Web Content fields:
+
+1. Open the Web Content application in a Site or Asset Library.
+
+1. Click *Actions* (![Actions Button]()) for the content you want to translate and select *Translate*.
```
Comment (jrhoun) [inline docs/dxp/latest/en/content-authoring-and-management/web-content/translating-web-content/automatically-translating-web-content.md:25]: @JamesAGarcia Missing icon.

File: docs/dxp/latest/en/content-authoring-and-management/web-content/translating-web-content/automatically-translating-web-content.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+   ![Open the Web Content translation interface.](./automatically-translating-web-content/images/02.png)
+
+1. Click the top left *Auto Translate* button to translate all Web Content fields into the selected language.
+
+   ![Click the top left Auto Translate button to translate all Web Content fields.](./automatically-translating-web-content/images/03.png)
+
+   Alternatively, click a field's individual *Auto Translate* button (![Auto Translate Button]()) to only translate its text.
```
Comment (jrhoun) [inline docs/dxp/latest/en/content-authoring-and-management/web-content/translating-web-content/automatically-translating-web-content.md:33]: @JamesAGarcia Missing icon.

**Comment by JamesAGarcia** [comment]
@jrhoun, I've fixed the links and added the new translate icon. They are in commit df7811b here: https://github.com/sez11a/liferay-learn/pull/524 I wasn't sure of the best way to get them to you.

**Comment by jrhoun** [comment]
Superseded by https://github.com/jrhoun/liferay-learn/pull/1107

**Pull request closed without merging**